docs/en/status-communities/token-based-access-to-communities-and-channels.md

```diff
@@ -0,0 +1,59 @@
+---
+id: 653
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Token-based access to communities and channels
+
+!!! tip ""
+    Currently, you can only configure token-based access using Status desktop.
+
+In the Status app, community [owners and admins][permissions-by-role-in-status-communities] can use token-based permissions to organize and control their communities and channels.
+
+As a community owner, you can configure token-based permission in [the community as a whole][set-up-your-community-permissions] and individual [channels within it][set-up-channel-permissions]. This functionality allows you to set up open communities with specific token-gated channels or [entirely closed communities][set-up-a-private-community] that require certain tokens for access.
+
+!!! note
+    Both community owners and admins can define token-based permission using Status desktop. For more information, check out [Permissions by role in Status Communities][permissions-by-role-in-status-communities].
+
+## Define and assign token-based access
+
+To define different access levels in your community, you start with a token.
+
+- [x] Choose one or more tokens you want to use. This can be an existing token (such as ETH, SNT or an ENS name), [a new token you mint][mint-tokens-for-your-community] or [a token you import][import-tokens-to-your-community] into the community.
```

Comment:
'This can be an existing token': could we replace 'token' with 'asset'? The terminology I'm using for the Wallet explicitly differentiates between ETH and 'tokens', just so that it's easier for users to understand why they're asked to pay network fees in ETH when sending e.g. DAI. So I think it'd be better to avoid referring to ETH as a 'token', otherwise users can get confused about the terms we're using - I hope that makes sense!

Reply:
We can't replace 'token' with 'asset'. In our app's terminology, 'token' includes assets, collectibles, and ENS names. The owner can use any of these three (or a combination) to set up permissions.

Therefore, I can't say, 'This can be an existing asset' because then, the owner will think s/he can only use assets, but the app gives them the option to use collectibles and ENS names as well (see the screenshot from the desktop app). Let me know if I need clarification.

![Screenshot 2023-06-28 at 3 39 57@2x](https://github.com/status-im/help.status.im/assets/62282406/f70ecabb-d7e4-4dd9-a046-a09a9c046040)

Reply:
I replaced 'ETH' with 'DAI' in the example.